config-linux.md

```diff
@@ -625,6 +625,22 @@ The following parameters can be specified to set up seccomp:
     * `SECCOMP_FILTER_FLAG_LOG`
     * `SECCOMP_FILTER_FLAG_SPEC_ALLOW`
 
+* **`listenerPath`** *(string, OPTIONAL)* - specifies the path of UNIX domain socket over which the runtime will send the [container process state](#containerprocessstate) data structure when the `SCMP_ACT_NOTIFY` action is used.
+    This socket MUST use `AF_UNIX` domain and `SOCK_STREAM` type.
```

Comment:
Wait, why not SOCK_SEQPACKET, or DGRAM?

Reply:
@sargun marking as solved as you recently reviwed with a LGTM. Please feel free to reopen if you think this isn't addressed :)